truss/templates/vllm/vllm.Dockerfile.jinja

```diff
@@ -1,4 +1,4 @@
-FROM baseten/vllm:v0.1
+FROM baseten/vllm:v0.2
```

Comment:
can we move the Dockerfile for generating this base image so that we can keep track of those changes as well?